dip-0011.md

```diff
@@ -166,21 +170,21 @@ Applications should encourage end users to maintain a healthy balance of
 credits for their Identities.
 
 As more applications come into existence, keys that are only used for a specific
-application, a group of applications (e.g., from a single developer) or to
+application, a group of applications (e.g. from a single developer) or to
```

Comment:
Necessity of a comma here is debatable (dependent on which style guide one uses). If changing, it must be applied consistently through the doc.

Reply:
I don't think anywhere else in the doc has this kind of a comma

Reply:
https://github.com/dashpay/dips/pull/66/commits/c7963e007017da552fe5cdbb3a55ae531fea7cdf

Reply:
:+1: Agree, I only see one other eg and it doesn't have one.